<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenCapture</class>
 <widget class="QWidget" name="ScreenCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File Name Root:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="fileNameBtn">
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="jpgImage">
      <attribute name="title">
       <string>JPG Image</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Capture Qualiy:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="jpgQualitySlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="jpgQualityLabel">
           <property name="text">
            <string> %</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Single Image:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="jpgScreenCapBtn">
            <property name="text">
             <string>Screen Capture Now!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Image Series:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Capture Interval:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="jpgIntervalSlider">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="pageStep">
               <number>50</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="jpgIntervalLabel">
              <property name="text">
               <string>msec.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="jpgStartBtn">
              <property name="text">
               <string>Start</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="jpgPauseBtn">
              <property name="text">
               <string>Pause</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="jpgStopBtn">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Images Taken:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="jpgImgsTaken">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bmpImage">
      <attribute name="title">
       <string>BMP Image</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Single Image</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="bmpScreenCapBtn">
            <property name="text">
             <string>Screen Capture Now!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Image Series</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Capture Interval:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="bmpIntervalSlider">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="pageStep">
               <number>50</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="bmpIntervalLabel">
              <property name="text">
               <string>msec.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="bmpStartBtn">
              <property name="text">
               <string>Start</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bmpPauseBtn">
              <property name="text">
               <string>Pause</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bmpStopBtn">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Images Taken:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bmpImgsTaken">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aviMovie">
      <attribute name="title">
       <string>AVI Movie</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Record Movie</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Frame Interval:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="aviIntervalSlider">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="pageStep">
               <number>50</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="aviIntervalLabel">
              <property name="text">
               <string>msec.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="aviStartBtn">
              <property name="text">
               <string>Start</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="aviPauseBtn">
              <property name="text">
               <string>Pause</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="aviStopBtn">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Images Taken:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="aviImgsTaken">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
